lisa/transformers/kernel_source_installer.py

```diff
@@ -77,6 +76,20 @@ class SourceInstallerSchema(BaseInstallerSchema):
     # Steps to modify code by patches and others.
     modifier: List[BaseModifierSchema] = field(default_factory=list)
 
+    # This is absolute path where kernel config file is located
```

Comment:
If the config is in-tree, then users need to know where lisa is cloning the tree to be able to specify the absolute path. Users should not need to figure this out.

Reply:
@anirudhrb Could you please give explain

Reply:
@anirudhrb As we discussed, kernel config file will be relative path to the kernel source code path